createUiDefinition.json

```diff
@@ -72,7 +70,7 @@
                         "constraints": {
                             "required": true,
                             "regex": "^(?=.*[A-Z])(?=.*[.!@#$%^&()\\-_=+])(?=.*[0-9])(?=.*[a-z]).{12,40}$",
-                            "validationMessage": ""
+                            "validationMessage": "The Password must be 12-40 characters long and contain at least one uppercase letter, digit and special character .!@#$%^&()-_=+"
```

Comment:
Hi Abika, I think the regex here means: 

The password must contain 12 to 40 characters, including at least one uppercase letter, one lowercase letter, one digit, and one special character from .!@#$%^&()\\-_=+

Reply:
Hi Vanessa, I have just added the descriptive sentence which is already present in the template as a validation message, I can look into this " positive lookahead - http://www.regular-expressions.info/lookaround.html " and get back to you

Reply:
Hi Abika, what I meant was: I think the case "including at least one lowercase letter" is missing from your original sentence which is a requirement from the regex expression "(?=.*[a-z])"

Reply:
yes, you are right that is missing. But I am unsure if we have to be this restrictive.

Reply:
I'm not sure about the requirements, but if we do not need to restrict on lower case letters then the regex expression needs to be modified as well.